lib/philomena_web/templates/image/related/index.html.slime

```diff
@@ -2,9 +2,10 @@
   .block__header
     .block__header__title.hide-mobile
       ' Viewing related images for
-      strong>
-        | #
-        = @image.id
+      = link to: ~p"/images/#{@image}" do
+          strong>
+          | image #
```

Comment:
wrong indentation, needs to be 1 level  higher. Right now this would render as
```
<a href="..."><strong> </strong>image #1234</a>
```

Reply:
Header links are already bold so removed it